src/components/ShowResultsSidebar.js

```diff
@@ -12,37 +13,42 @@ function ShowAttendees({ partitionedRespondents, time }) {
   // logged in, has admin rights, and whether the respondent has responded
   function renderRespondent(responder, respondent, attending) {
     const displayName = respondent.user ? respondent.user.name : respondent.anonymousName;
+
     return (
-      <div
-        className={classnames(
-          styles.respondents,
-          attending ? styles.borderAccept : styles.borderReject,
-        )}
-        key={responder}
-      >
-        {displayName}
-        <MaterialIcon icon="visibility" className={styles.icon} onClick={() => {}} />
-      </div>
+      <ListItem>
+        <ListItemGraphic graphic={<MaterialIcon icon='account_circle'/>} />
+        <ListItemText
+          className={styles.listText}
+          primaryText={displayName}
+          secondaryText={respondent.user ? respondent.user.email : 'Anonymous User'}
```

Comment:
I'm not sure if "Anonymous User" will be confusing. The user has a name. In the response page, we said that people could "Respond with a pseudonym", but I don't think pseudonyms are the right word as well. It may be best just not to say anything